firefox_ui_harness/mixins/updates.py

```diff
@@ -0,0 +1,80 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from base import FirefoxUIOptions, FirefoxUITestRunner
+from ..firefox_ui_test import FirefoxUITestCase
+
+import firefox_ui_tests
+
+
+class UpdateOptions(FirefoxUIOptions):
```

Comment:
This is a linear inheritance hierarchy. Calling these mixins doesn't make sense to me.